xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/java/org/xwiki/refactoring/script/RefactoringScriptService.java

```diff
@@ -542,4 +546,15 @@ public Job replaceUser(ReplaceUserRequest request)
     {
         return this.execute(RefactoringJobs.REPLACE_USER, request);
     }
+
+    /**
+     *
+     * @return {@code true} if the configuration allows to skip the recyclebin and if the current userhas the right
+     * to do so (ie, the current user is advanced).
+     * @since 12.8RC1
+     */
+    @Unstable
+    public boolean canSkipRecycleBin() {
```

Comment:
Same comment as above, why not be consistent and make it positive?

Reply:
Method renamed `isAllowedToSkipTheRecycleBin`. (see https://github.com/xwiki/xwiki-platform/pull/1369/files#diff-5dddd4cfe21902963054e8f43d04d0feR557)